liquibase-maven-plugin/src/main/java/org/liquibase/maven/plugins/AbstractLiquibaseMojo.java

```diff
@@ -1176,6 +1193,11 @@ private Object getDefaultValue(Field field) throws IllegalAccessException {
     }
 
     private void setFieldValue(Field field, String value) throws IllegalAccessException {
+        try {
+            value = Scope.getCurrentScope().getSingleton(ConfiguredValueModifierFactory.class).override(value);
+        } catch (Exception e) {
+            getLog().warn("Failed to modify value for field " + field.getName(), e);
+        }
```

Comment:
@filipelautert Should I be catching the exception here? I think not, like if someone uses an `aws-secrets` secrets vault but doesn't have their AWS creds set up, the execution should fail, just like in the CLI.

Reply:
If you don't catch it here, what would be the impact for the current behavior of the plugin? If none then just remove it :D

Reply:
Thinking about it, there would be no impact. It should definitely be removed. I'll do that.